src/components/App/App.js

```diff
@@ -52,17 +55,23 @@ function App() {
 
 
   return (
-    <div className="App">Salon
-
-      <Wall artworks={wall} />
+    <div className="App">
+      <Route 
+        exact path="/"
+        render={() => <Wall artworks={wall} />}
+      />
+      <Route exact path='/:artPieceID' render={({ match }) => {
+        const { artPieceID } = match.params;
```

Comment:
Glad you updated this - I feel like match can still be used when we get into context usage.  Haven't seen it in practice but I feel like it should be good to work with.